test/intl402/String/prototype/toLocaleUpperCase/special_casing_Lithuanian.js

```diff
@@ -0,0 +1,188 @@
+// Copyright (C) 2015 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Check if String.prototype.toLocaleUpperCase supports language-sensitive mappings defined in SpecialCasings (Lithuanian)
+info: >
+    The result must be derived according to the case mappings in the Unicode character database (this explicitly
+    includes not only the UnicodeData.txt file, but also the SpecialCasings.txt file that accompanies it).
+es5id: 15.5.4.16
+es6id: 21.1.3.21
+---*/
+
+// SpecialCasing.txt, conditional, language-sensitive mappings (Lithuanian).
+
+// COMBINING DOT ABOVE (U+0307) not removed when uppercasing capital I and J.
+assert.sameValue(
+  "I\u0307".toLocaleUpperCase("lt"),
+  "I\u0307",
+  "COMBINING DOT ABOVE preceded by LATIN CAPITAL LETTER I"
+);
+assert.sameValue(
+  "J\u0307".toLocaleUpperCase("lt"),
+  "J\u0307",
+  "COMBINING DOT ABOVE preceded by LATIN CAPITAL LETTER J"
+);
+
+
+// Code points with Soft_Dotted property (Unicode 5.1, PropList.txt)
+var Soft_Dotted = [
```

Comment:
Even though the variable name `Soft_Dotted` matches what's in the Unicode spec, we should change it to `softDotted` to keep the variable naming consistent across the project.